data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -138,7 +138,7 @@
     "(?i).*PROCTER.*&.*GAMBLE": "P&G",
     "(?i)P.*&.*G": "P&G",
     "(?i).*RECKITT.*": "RECKITT BENCKISER",
-    "(?i)^RB$": "RECKITT BENCKISER",
+    "(?i)^RB$": "RECKITT BENCKISER","(?i)^RB .*": "RECKITT BENCKISER",
```

Comment:
You don't want to relax regular expression like this. 'RB' (upper or lowercase) could appear anywhere for OTHER advertisers as well. Then this regular expression, "(?i)^RB .*", will cause incorrect mappings.